### DISCUSSION TITLE
PointLight Attachment

### DISCUSSION THREAD

**Discussion by andrejhronco** [cables-gl/cables_docs#173] [Feature Requests] [answered]
There used to be an `attachment` port on the PointLight Op a couple of years ago and it looks like it got removed (intentionally | unintentionally). There's an [issue](https://forum.cables.gl/t/new-pointlight-object-position-output/699) on the old forum where `simod` said that it would be added back in a future update, but it never happened.

Is there another way of getting the same functionality without the attachment if it's not considered needed/useful in the PointLight Op?

**Comment by pandrr** [comment]
it basically was just an extra trigger where you could attach a sphere or something like that.

**Comment by pandrr** [comment, marked as answer]
actually i was wrong, what it did was internally translate to the light position.

to reproduce this:

1. attach transform unter pointlight
2. set coordinates to same point
3. add sphere, or whaterver you want to show there

**Reply by andrejhronco** [reply]
Thanks Pandur, that makes sense.
